src/vivarium_public_health/population/base_population.py

```diff
@@ -138,20 +160,20 @@ def on_time_step(self, event: Event) -> None:
         population["age"] += utilities.to_years(event.step_size)
         self.population_view.update(population)
 
+    ##################
+    # Helper methods #
+    ##################
+
     @staticmethod
-    def select_sub_population_data(
-        reference_population_data: pd.DataFrame,
-        year: int,
+    def get_demographic_proportions_for_creation_time(
```

Comment:
Is this a candidate for longest method name? It's WAY clearer though!

Reply:
Agreed it's much more obvious what's happening now.